features-json/colr-v1.json

```diff
@@ -0,0 +1,478 @@
+{
+  "title":"COLR/CPAL(v1) Font Formats",
+  "description":"COLRv1 is an improved version of COLRv0, this is also part of the OpenType specification. COLRv1 supports additional graphic capabilities. In addition to solid colors, gradient fills can be used, as well as more complex fills using other graphic operations, including affine transformations and various blending modes.",
+  "spec":"https://docs.microsoft.com/en-us/typography/opentype/spec/colr",
+  "status":"other",
+  "links":[
+    {
+      "url":"https://developer.chrome.com/blog/colrv1-fonts/",
+      "title":"COLRv1 Color Gradient Vector Fonts in Chrome 98"
+    },
+    {
+      "url":"https://github.com/googlefonts/color-fonts",
+      "title":"COLRv1 version of Noto Emoji"
+    },
+    {
+      "url":"https://pixelambacht.nl/chromacheck/",
+      "title":"A tool to check whether the browser supports OpenType color formats"
+    },
+    {
+      "url":"https://www.colorfonts.wtf/#w-node-85d8080e63a6-0134536f",
+      "title":"Where can I use color fonts"
+    }
+  ],
+  "bugs":[
+    
+  ],
+  "categories":[
+    "Other"
+  ],
+  "stats":{
+    "ie":{
+      "5.5":"n",
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"n",
+      "10":"n",
+      "11":"n"
+    },
+    "edge":{
+      "12":"y",
+      "13":"y",
+      "14":"y",
+      "15":"y",
+      "16":"y",
+      "17":"y",
+      "18":"y",
```

Comment:
That's probably not true, is it?

Reply:
My mistake, COLRv1 is not supported here.